src/libs/actions/IOU.ts

```diff
@@ -759,8 +759,8 @@ function getMoneyRequestInformation(
         // If the scheduled submit is turned off on the policy, user needs to manually submit the report which is indicated by GBR in LHN
         needsToBeManuallySubmitted = isFromPaidPolicy && !policy?.harvesting?.enabled;
 
-        // If the linked expense report on paid policy is not draft, we need to create a new draft expense report
-        if (iouReport && isFromPaidPolicy && !ReportUtils.isDraftExpenseReport(iouReport)) {
+        // If the linked expense report on paid policy is not draft and not instantly submitted, we need to create a new draft expense report
+        if (iouReport && isFromPaidPolicy && !ReportUtils.isDraftExpenseReport(iouReport) && !ReportUtils.isInstantSubmittedState(iouReport)) {
```

Comment:
Hmm this scares me a little bit... Like are we sure "we need to create a new draft expense report" in this case? I think no, because if we're requesting money on an expense report on an instant submit policy, we should be creating a **submitted** report (b/c "instant submit"), not a **draft** report 🤔 

Do we have a manual test that covers this exact case? If so, are we sure this is working correctly?

Reply:
This is working correctly as Step `1` of our test case involves creating a new expense report.  

But I think creating a new expense report is a little tricky too. In online scenarios, the BE would send back the `submitted` state and it is all good in FE. But, in offline scenarios with `instant` reporting frequency it does not seem right to create a new draft expense report. However, it also does not seem right to optimistically generate a `submitted` report as the `instant` reporting frequency may have been changed to something else in BE when the user is offline. So, ideally, it may be better to wait for the user to come online and let BE decide the right thing to do. Maybe the way to resolve this is to hide the `Submit` button in the report preview and also in the expense report header in offline scenarios when it is `instant` reporting frequency. Or maybe hide the `submit` buttons at all times for `instant` reporting frequency. 

But this looks like a separate issue and may need some more thought through. Do you think creating a separate issue for this is better? We have already extended this issue’s scope with the handling of delete money request issues.
cc @Beamanator

Reply:
> Maybe the way to resolve this is to hide the `Submit` button in the report preview and also in the expense report header in offline scenarios when it is `instant` reporting frequency. Or maybe hide the `submit` buttons at all times for `instant` reporting frequency.

This should be the case - Are we currently able to see the `Submit` button when the auto reporting frequency is `instant`? If so, please let me know and I'll definitely make a new bug report for that :D 👍

Reply:
For the online / offline cases...
1. If we know (before going offline) that the auto reporting frequency is `instant`, we DO need to optimistically auto-submit the report - we want to assume that the front & backends are sync'd, the "weird" scenario would be if they aren't.
2. Do you know exactly how this code in this line is hit? Is it really only when we're offline? I haven't taken the time to check but I'm happy to if you're not 100% certain 👍

Reply:
Agreed - and it looks like this is actually already being taken care of in https://github.com/Expensify/App/pull/36388/files#diff-577fcc5f3a5e4930916ff310e61c38edfa093792fe3ab55b9a30d2aecd8811db